src/index.js

```diff
@@ -93,3 +98,31 @@ allToDoInputs.forEach((a, i, arr) => {
     toNodeList(copyArr);
   });
 });
+
+// checkboxes
+const checkboxes = document.querySelectorAll('.before');
+
+checkboxes.forEach((a, i) => {
+  a.addEventListener('click', () => {
+    if (a.parentElement.classList.contains('completed')) {
+      a.parentElement.classList.remove('completed');
+      const currentLocalList = JSON.parse(localStorage.getItem('activityArr'));
+      currentLocalList[i].completed = false;
+      localStorage.setItem('activityArr', JSON.stringify(currentLocalList));
+      window.location.reload();
```

Comment:
[OPTIONAL] Great job creating your to-do list, well done! However, this `window.location.reload();` is making the app reload far many times, it's reloading for every click, it would be great if you had a function that updates the displayed data instead of reloading the app every time, this way your app would be faster and more efficient.